lib/tag-field/index.tsx

```diff
@@ -282,6 +293,12 @@ export class TagField extends Component<Props, OwnState> {
   }
 }
 
-export default connect(null, { updateNoteTags } as S.MapDispatch<
-  DispatchProps
->)(TagField);
+const mapStateToProps: S.MapState<StateProps> = ({
+  settings: { keyboardShortcuts },
+}) => ({
+  keyboardShortcuts,
+});
+
+export default connect(mapStateToProps, {
+  updateNoteTags,
+} as S.MapDispatch<DispatchProps>)(TagField);
```

Comment:
Fun fact: this is the only place in the codebase we're doing this (as opposed to defining a separate `MapDispatchToProps` variable) and I don't think I like it. I do realize it was already like that, though.

Reply:
I'll do a follow up